lib/js_routes.rb

```diff
@@ -36,6 +36,7 @@ class JsRoutes
 
   LAST_OPTIONS_KEY = "options".freeze
   FILTERED_DEFAULT_PARTS = [:controller, :action, :subdomain]
+  URL_OPTIONS = [:protocol, :domain, :host, :port]
```

Comment:
No, I don't mean that. Please add a test that test for inclusion of subdomain in `URL_OPTIONS` if it exists.